dags/orbax/maxtext_mtc_emergency_save_local.py

```diff
@@ -0,0 +1,190 @@
+"""
+A DAG to run MaxText multi-tier and emergency checkpointing tests.
+
+This DAG performs a series of tests to save and validate checkpoints
+for the MaxText model. It runs tests in two modes: one with the replicator
+service enabled (Multi-tier Checkpointing) and one without
+(Emergency Checkpointing). The tests are executed on a TPU multi-pod cluster.
+
+Variables:
+  SCHEDULE (str): The cron schedule.
+  DAG_TEST_NAME (str): Unique DAG name.
+  BASE_OUTPUT_DIRECTORY (str): GCS path for saved checkpoints.
+  DOCKER_IMAGES (list): Tuples of setup mode and Docker image.
+  RAM_DISK (str): Local directory mounted on each machine.
+  TEST_CONFIGS (dict): Accelerator type and slice count.
+  CLUSTERS (dict): Maps accelerator to TPU cluster configs.
+  STEP (int): Total training steps.
+  LOCAL_CHECKPOINT_PERIOD (int): Frequency (in steps) for local checkpoints.
+  REPLICATOR_BACKUP_INTERVAL_MINUTES (int): Interval (in minutes) for replicator.
+  NAME_PREFIX (str): Prefix for test run names.
+  MODEL_NAME (str): Name of the model under test.
+"""
+
+import datetime
+from dataclasses import asdict
+
+from airflow import models
+from airflow.utils.task_group import TaskGroup
+
+from dags import composer_env, gcs_bucket
+from dags.common import test_owner
+from dags.common.vm_resource import DockerImage, XpkClusters
+from dags.multipod.configs import gke_config
+from dags.multipod.configs.common import SetupMode
+from dags.orbax.util import logging_mtc as log
+from dags.orbax.util import multi_tier_checkpoint_util as mtc
+from xlml.utils.xpk import BRANCH_ABHINAV_MTC
+from absl import logging
+
+SCHEDULE = "0 10 * * *" if composer_env.is_prod_env() else None
+DAG_TEST_NAME = "maxtext_emc_and_mtc_orbax_save_local"
+BASE_OUTPUT_DIRECTORY = (
+    f"{gcs_bucket.MTC_AUTOMATION_BUCKET}/"
+)
```

Comment:
```suggestion
BASE_OUTPUT_DIR = gcs_bucket.MTC_AUTOMATION_BUCKET
```

and then 

`OUT_DIR = f"{BASE_OUTPUT_DIR}{folder}"` -> `OUT_DIR = posixpath.join(BASE_OUTPUT_DIR, folder)`

Reply:
Done. Please check new commit.